### DISCUSSION TITLE
패키지 내 라이브러리 구조를 정리 할 수 있는 방법이 있을까요?

### DISCUSSION THREAD

**Discussion by topkim993** [nsoojin/MiniSuperApp-fastcampus#14] [아무 질문이나 환영합니다] [answered]
<img width="1003" alt="스크린샷 2022-06-02 오후 12 51 52" src="https://user-images.githubusercontent.com/57055565/171549641-e78fe869-bffa-4fa4-8073-15072fe6bed5.png">
위 이미지와 같이 묶어 주고 싶은데요. 혹시 방법이 있을까 해서 질문 드립니다.

**Comment by nsoojin** [comment, marked as answer]
안녕하세요. target에 path를 지정해주시면 됩니다.
```
.target(
      name: "AddPaymentMethod",
      dependencies: [
      ],
      path: "AddPaymentMethod/Interface"
    ),
```

각각 아래처럼 해주시면 되겠죠.
path: "AddPaymentMethod/Interface"
path: "AddPaymentMethod/Implement"
path: "AddPaymentMethod/TestSupport"

**Reply by topkim993** [reply]
답변 감사 드립니다. 많은 도움 되었어요!
